js/particle-modeler/index.js

```diff
@@ -111,9 +118,35 @@ export default class Interactive extends PureComponent {
         newState["showNewAtom"+el] = false;
         _this.setState(newState);
         _this.addNewDraggableAtom(el);
+      } else {
+        if (d.x > 4.187 && d.x < 4.328 && d.y > 0.141 && d.y < 0.287) {
```

Comment:
Surprisingly hard-coded - is there a way we can centralize these locations so that the delete icon can be repositioned more easily?